### ISSUE TITLE
[Adaptive Cards - "InputswithValidation.JSON"]: “Reply” button looks like a normal text on card.

### ISSUE THREAD

**Issue by HasnuddinMohammed** [microsoft/AdaptiveCards#8742]
### Target Platforms

UWP

### SDK Version

1.2.2310.16007

### Application Name

Adaptive Card Visualizer

### Problem Description

**Test Environment:** 
Platform: Win32 
OS version: 23H2 (Build 25967.1010) 
Application: Adaptive Card Visualizer 
Application Version: 1.2.2310.16007 

**Repro steps:​** 
1. Open the Adaptive Card Visualizer application. 
2. Navigate the "InputswithValidation.JSON card. 
3. Navigate the input text edit fields and observe the issue 
 
**Actual Result:**    
Low vision users found that, “Reply” button looks like a normal text.

**Expected result:**  
To make the difference between the controls and normal text, solid boarders should be provided for the “reply” button. 

**User Experience:**  
Users are unable to identify the difference between the controls and plain text if no boundaries are defined. 

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/79564581/8ec8a6be-1f89-4a5c-8303-589b0406c630

### Card JSON

```JSON
NA
```
